pipeline/transformer/transformer_sierra/src/test/scala/weco/pipeline/transformer/sierra/transformers/subjects/SierraOrganisationSubjectsTest.scala

```diff
@@ -139,7 +139,7 @@ class SierraOrganisationSubjectsTest
       subject.id shouldBe IdState.Identifiable(
         SourceIdentifier(
           identifierType = IdentifierType.LCNames,
-          ontologyType = "Subject",
+          ontologyType = "Organisation",
```

Comment:
This is a change to the original behaviour.  The change is "more correct", but I think we intend to get rid of this entirely, so if it just comes along for the ride, that's fine, but it will result in any organisation subject to be given a new canonical id.

https://wellcome.slack.com/archives/C02ANCYL90E/p1658140051620059?thread_ts=1658132806.695909&cid=C02ANCYL90E

https://wellcome.slack.com/archives/C02ANCYL90E/p1658148261259319?thread_ts=1658132806.695909&cid=C02ANCYL90E

Reply:
Ah, oops, then I’ll revert this change.

I think this will require a full reindex – it’ll also mint new source identifiers for every Sierra contributor.